app/views/products/index.html.erb

```diff
@@ -1,3 +1,8 @@
-<h1>Products#index</h1>
-<p>Find me in app/views/products/index.html.erb</p>
+<h1>Greta's Top Picks</h1>
+<% @products.each do |product| %>
+  <%= product.name %>
+  <%= product.desc %>
+  <%= product.price %>
+  <% end %>
```

Comment:
Please be mindful of code formatting

Reply:
I would use `<ul>` and `<li>` in this case to create a better structure. You can always remove it later when you add styling.

Reply:
Agree

Reply:
...and format code. Again